resource/reactivedrop_brazilian.txt

```diff
@@ -210,23 +210,23 @@
 "[english]asw_SynTekFlagLabel4"		"Turbine Failure"
 "asw_SynTekFlagLabel4"		"Falha na turbina"
 "[english]asw_SynTekFlagLabel5"		"Heat Ex. Overload"
-"asw_SynTekFlagLabel5"		"Sobrecarga do trocador de calor"
+"asw_SynTekFlagLabel5"		"Sobrecarga do Permutador de Calor"
 "[english]asw_SynTekFlagLabel6"		"Heat Exchanger Failure"
-"asw_SynTekFlagLabel6"		"Falha no trocador de calor"
+"asw_SynTekFlagLabel6"		"Falha no PErmutador de Calor"
```

Comment:
@FurtadoPires 
PErmutator?

Reply:
@Duke-M-It was a little typo, thank you.